CONTRIBUTING.md

```diff
@@ -160,5 +159,5 @@ For Web WorldWind tutorials and examples, please check out our website: https://
 
 For community support and FAQs, check out the WorldWind Forum: https://forum.worldwindcentral.com/.
 
-To reach our Administrative Contact, please email: [patrick.hogan@nasa.gov](mailto:patrick.hogan@nasa.gov).
+To reach our Administrative Contact, please email: [patrick.hogan@nasa.gov](arc-worldwind@mail.nasa.gov).
```

Comment:
Remove Patrick's name.

Reply:
It's fixed now. I added 'mailto:' to the links.